### ISSUE TITLE
Bug in /tools/geometry_utils.py line 132

### ISSUE THREAD

**Issue by rpartzsch** [SiLab-Bonn/beam_telescope_analysis#37]
In /tools/geometry_utils.py in the function get_line_intersection_with_plane is a small bug in line 132. One can just remove the raise. 

https://github.com/SiLab-Bonn/beam_telescope_analysis/blob/master/beam_telescope_analysis/tools/geometry_utils.py#L132C1-L132C14

**Comment by leloup314** [comment]
Yes, looks like a bug to me as well! Maybe @YannickDieter can confirm?

**Comment by YannickDieter** [comment]
What do you mean with "One can just remove the raise." 

To me the `raise` does not make much sense and we should remove it.

**Comment by leloup314** [comment]
He means exactly what you said: The `raise` statement should be removed since there is no exception to re-raise which will result in an exception itself.

**Comment by YannickDieter** [comment]
Feel free to fix that in any branch you are working with at the moment :) 
@rpartzsch
